### PR TITLE
Added xdebug extension

### PR DESCRIPTION
**Pull request by Padrio**

### CODE REVIEW

**Comment by Padrio** [comment]
How about giving some feedback or anything else? @eko

File: README.md [hunk 1]
```diff
@@ -64,6 +64,11 @@ You can access Nginx and Symfony application logs in the following directories o
 
 You can also use Kibana to visualize Nginx & Symfony logs by visiting `http://symfony.localhost:81`.
 
+# Use xdebug!
+
+To use xdebug change the line `"docker.host:127.0.0.1"` in docker-compose.yml and replace 127.0.0.1 with your machines ip addres.
```
Comment (eko) [inline README.md]: Great but can you fix typos here please:
`your machines ip addres` -> `your machine ip address`

File: README.md [hunk 2]
```diff
@@ -64,6 +64,11 @@ You can access Nginx and Symfony application logs in the following directories o
...
+If your IDEs default port is not set to 5902 you should do that, too.
```
Comment (eko) [inline README.md]: `IDEs` -> `IDE`

**Comment by Padrio** [comment]
I fixed the typos so it can be merged.

**Comment by eko** [comment]
Thank you @Padrio! I will just give it a try tomorrow and I will merge then, but it seems to be ok :)

**Comment by eko** [comment]
It's merged. Thank you @Padrio!

**Pull request merged**